index.bs

```diff
@@ -1994,93 +2099,208 @@ The [=remote end steps=] with |command parameters| are:
 
   1. Let |request| be a new [=/request=] whose URL is |url record|.
 
-  1. Let |navigation id| be the string representation of a
-     [[!RFC4122|UUID]] based on truly random, or pseudo-random numbers.
+  1. Return the result of [=await a navigation=] with |context|, |request|, and
+     |wait condition|.
+
+</div>
+
+#### The browsingContext.reload Command ####  {#command-browsingContext-reload}
```

Comment:
What's the browsingContext.reload addition doing here? I assume it'll go away with a rebase and will ignore here.